virtwho/virt/virt.py

```diff
@@ -708,15 +708,15 @@ def _send_data(self, data_to_send):
                         retry = False
                     except ManagerThrottleError as e:
                         if self._oneshot:
-                            self.logger.debug('429 encountered when sending virt guests in '
-                                              'oneshot mode, not retrying')
+                            self.logger.info('429 encountered when sending virt guests in '
```

Comment:
Is there a knowledge base article we can link to here?
The messaging is still a bit unclear if intended for an end user and not a dev.
I would suggest something like "Rate limit exceeded while sending host-guest mapping, please see {link to knowledge base article}" (optionally with the following appended in oneshot mode) " running in oneshot mode, not retrying".

Similar for the below log message as well.

Reply:
I found this article: https://access.redhat.com/solutions/2212941, but it would be necessary to modify example of error message in this article.